pkgs/data/fonts/iosevka/update-bin.sh

```diff
@@ -1,20 +1,20 @@
 #! /usr/bin/env nix-shell
 #! nix-shell -i bash -p curl jq
 
-set -e
+set -euo pipefail
+cd "$(dirname "${BASH_SOURCE[0]}")"
 
 release=$(curl -s https://api.github.com/repos/be5invis/Iosevka/releases/latest)
 
-oldVersion=$(nix-instantiate --eval -E 'with import ./. {}; lib.getVersion iosevka-bin' | tr -d '"')
+oldVersion=$(nix-instantiate --eval -E 'with import ../../../.. {}; lib.getVersion iosevka-bin' | tr -d '"')
```

Comment:
This seems wrong; you're supposed to run the update script from the root directory of nixpkgs.

Reply:
The script changes the directory of itself before executing these commands.

Reply:
It seems, at least, uncommon:

```
v@february ~/s/nixpkgs (local)> rg -t sh 'import \./' | wc -l
10
v@february ~/s/nixpkgs (local)> rg -t sh 'import \.\./' | wc -l
3
```